rootfs/Dockerfile

```diff
@@ -10,19 +10,20 @@ RUN adduser --system \
 COPY requirements.txt /app/requirements.txt
 
 RUN buildDeps='gcc git libffi-dev libpq-dev python3-dev'; \
-	apt-get update && \
-	apt-get install -y --no-install-recommends \
+    apt-get update && \
+    apt-get install -y --no-install-recommends \
         $buildDeps \
         libpq5 \
         python3 \
         sudo && \
     ln -s /usr/bin/python3 /usr/bin/python && \
-	curl -sSL https://bootstrap.pypa.io/get-pip.py | python - pip==8.1.2 && \
-	mkdir -p /configs && chown -R deis:deis /configs && \
-	pip install --disable-pip-version-check --no-cache-dir -r /app/requirements.txt && \
-	apt-get purge -y --auto-remove $buildDeps && \
-	apt-get clean && \
-	rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/* /usr/share/man /usr/share/doc
+    curl -sSL https://bootstrap.pypa.io/get-pip.py | python - pip==8.1.2 && \
+    mkdir -p /configs && chown -R deis:deis /configs && \
+    pip install --disable-pip-version-check --no-cache-dir -r /app/requirements.txt && \
```

Comment:
You'd think but if you drop onto the controller container and `dh -h` (or similar) that dir, it has data. Can you try it on yours just to confirm?